docs/_posts/kolia1985/2021-12-21-text_cleaner_v1_en.md

```diff
@@ -25,29 +25,109 @@ Model for cleaning image with text. It is based on text detection model with ext
 
 {:.btn-box}
 <button class="button button-orange" disabled>Live Demo</button>
-<button class="button button-orange" disabled>Open in Colab</button>
+[Open in Colab]([Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-ocr-workshop/blob/TrainingNotebooks/tutorials/Certification_Trainings/others/SparkOcrImageCleaner.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}){:.button.button-orange.button-orange-trans.co.button-icon}
```

Comment:
Colab notebook link is broken. Could you please check?

Reply:
solved